src/suggestions.js

```diff
@@ -59,13 +60,21 @@ export default class SimpleteSuggestions extends HTMLElement {
 			let selector = container && container.getAttribute(attr);
 			this[prop] = selector || DEFAULTS[prop];
 		});
+
+		// set aria roles for each option and ensure that it has a valid HTML id
+		let items = find(this, this.itemSelector);
+		items.forEach(el => {
+			el.setAttribute("role", "option");
+			el.id = el.id || "simplete-suggestion" + nid();
+			el.setAttribute("aria-selected", "false");
+		});
```

Comment:
minor details:

```suggestion
		find(this, this.itemSelector).forEach(el => {
			el.id = el.id || "simplete-suggestion" + nid();
			el.setAttribute("role", "option");
			el.setAttribute("aria-selected", "false");
		});
```

(not sure whether it's worth avoiding redundant `#id` assignments using `if`; probably doesn't make a difference)